symlink/fs_unix_test.go

```diff
@@ -405,3 +355,12 @@ func TestFollowSymlinkNoLexicalCleaning(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func mkTempDir(t *testing.T) (string, func()) {
```

Comment:
go 1.15 adds `(*testing.TB).TempDir()` which would be easier to use and won't require a cleanup.

But we'll have to wait until at least go 1.17 to use it :-\

Reply:
Ah, nice; didn't know about the `TempDir()`, but we can start using `CleanUp()`. I did a PR for that in https://github.com/moby/term/pull/21. That repo was still testing against go 1.13 for backward compatibility, but I see this repo only tests go 1.14 and go 1.15, so I can make a similar change here